crates/loro/src/lib.rs

```diff
@@ -1461,6 +1461,22 @@ impl LoroTree {
         self.handler.get_deep_value()
     }
 
+    /// When we delete a node, it is moved to a delete node named `TRASH` to represent deletion.
+    /// In the long run, there will be many tombstones of nodes that are no longer used under `TRASH`.
+    /// You can use `empty_trash()` to clean up these tombstones in `State`.
+    /// We still keep the relevant information in the Op history, checkout to the any version will still work.
+    ///
+    /// ## Some Notes:
+    /// We recommend that you use this method with caution and only when you think you have enough historical operations.
+    /// Because if you empty the trash, and there are some concurrent operations that moving the deleted nodes back
+    /// to the tree. These operations may not have an effect as they have been completely removed from the State.
+    ///
+    /// To avoid unnecessary memory and storage usage for this Operation, you need to ensure that it is
+    /// only called once in potentially concurrent operations.
+    pub fn empty_trash(&self) -> LoroResult<()> {
```

Comment:
I think we need a param to allow users to only empty trash that is old enough. 

For example, by using the lamport value, or the timestamp, or allowing users to decide the maximal items allowed in the trash

Reply:
Yes, it's better.

Reply:
Done.   For now, I just added `max_lamport`, if more parameters are needed, add them when necessary.